### PR TITLE
DAS-2165 - update to version number and minor updates based on PR feedback

### PR DESCRIPTION
**Pull request by sudha-murthy**
## Description

- version number updated to 2.3.0 from 2.2.2 and a minor update to the argument type annotation

## Jira Issue ID
DAS-2165

## Local Test Steps
git clone https://github.com/nasa/earthdata-varinfo.git
conda activate earthdata-varinfo
cd earthdata-varinfo
make develop
make test

## PR Acceptance Checklist
* [X] Jira ticket acceptance criteria met.
* [X] `CHANGELOG.md` updated to include high level summary of PR changes.
* [X] `VERSION` updated if publishing a release.
* [X] Tests added/updated and passing.
* [X] Documentation updated (if needed).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
-## v2.2.2
-### 2024-07-16
+## v2.3.0
+### 2024-07-17
```
Comment (flamingbear) [inline CHANGELOG.md:3]: This was already released so you'll need a new version above rather than deleting this one.

**Review by flamingbear — CHANGES_REQUESTED** [review]
Need to fix the changelog. But seems like you did the changes that @owenlittlejohns requested.

**Comment by owenlittlejohns** [comment]
Just reiterating my comments on the PR that will be closed:

- Because these changes were merged with a patch version rather than a minor version, and therefore released to PyPI, it seems a bit overkill to re-release them under a new version (even though that would get the correct version number).
- I am working on a PR that can include the type hint change in the next couple of days.

I think what I am saying is that we just stick with what we've got and don't make things more confusing with a duplicate release.

File: CHANGELOG.md (hunk 1 above)
Comment (sudha-murthy) [reply CHANGELOG.md:3]: will not do any updates. will let Owen do the next update including the version number
This PR can be closed out as well

**Pull request closed without merging**